src/FileParsers.pm

```diff
@@ -363,6 +372,8 @@ sub parseDataFile_lc {
 		next if ($line =~ /^#/);
 		$line =~ s/[\r\n]//g;
 		$line =~ s/\s+$//g;
+		$line =~ s/#.*// if ($file eq "pickupitems.txt" or $file eq "arrowcraft.txt");
```

Comment:
i think that it can be interesting to trim down whitespace before remove comments
thinking well, the code removed whitespaces at the end of line, and them remove comments
what if there is a space before the `#` ?
problably it will not affect any parsing, but we should check that

Reply:
Fair enough